--- libsrtp-2.8.0/test/util.c.orig	2026-03-14 10:49:34.000000000 +0100
+++ libsrtp-2.8.0/test/util.c	2026-06-25 20:56:12.076698684 +0200
@@ -55,7 +55,7 @@
 #include <stdio.h>
 
 /* include space for null terminator */
-static char bit_string[MAX_PRINT_STRING_LEN + 1];
+static char test_bit_string[MAX_PRINT_STRING_LEN + 1];
 
 #define ERR_STATUS_STRING(STATUS)                                              \
     case srtp_err_status_##STATUS:                                             \
@@ -266,11 +266,11 @@ char *octet_string_hex_string(const void
     }
 
     for (i = 0; i < length; i += 2) {
-        bit_string[i] = nibble_to_hex_char(*str >> 4);
-        bit_string[i + 1] = nibble_to_hex_char(*str++ & 0xF);
+        test_bit_string[i] = nibble_to_hex_char(*str >> 4);
+        test_bit_string[i + 1] = nibble_to_hex_char(*str++ & 0xF);
     }
-    bit_string[i] = 0; /* null terminate string */
-    return bit_string;
+    test_bit_string[i] = 0; /* null terminate string */
+    return test_bit_string;
 }
 
 static const char b64chars[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
